### DISCUSSION TITLE
Plans to incorporate IEEE 11073 into open mHeath?

### DISCUSSION THREAD

**Discussion by haydenrandles** [openmhealth/schemas#25] [General]
I was just wondering whether there was any intention to incorporate IEEE 11073 into the schemas in Open mHealth e.g. IEEE 11073-10206-2022

Cheers!

**Comment by scarini** [comment]
The Open mHealth schema repository is currently being updated to reference [IEEE 1752.1 schemas](https://opensource.ieee.org/omh/1752/-/tree/main) wherever relevant, namely schemas modeling sleep, physical activity measures, metadata and a number of utility schemas. [IEEE 1752.1](https://ieeexplore.ieee.org/document/9540821) Standard for Open Mobile Health Data: Representation of Metadata, Sleep, and Physical Activity Measures comprises the document and the schemas. The latter constitute the normative portion of the standard and are available on the IEEE open-source site. 
The Open mHealth-initiated work continues with [IEEE P1752.2](https://standards.ieee.org/project/1752_2.html), Standard for Mobile Health Data: Representation of Cardiovascular, Respiratory, and Metabolic Measures. The [IEEE P1752 Working Group website](https://sagroups.ieee.org/1752/) includes information on the ongoing activities.
